docs/gro-ontology.rst

```diff
@@ -52,7 +52,7 @@ And, in the opposite direction, the following will return a list of regions "Cal
 
   client.lookup_belongs('regions', 13055)
 
-`See this Getting Started section for navigation of relationships between entities. <./searching-data.html#lookup-contains>`
+See this Getting Started section for navigation of relationships between entities. `Searching Data#lookup-contains <searching-data#lookup-contains>`_
```

Comment:
Similar to above comment:

`See this Getting Started section for navigation of relationships between entities.<searching-data#lookup-contains>`

The user doesn't need to see the Searching Data#lookup-contains  text

Reply:
![image](https://user-images.githubusercontent.com/54555734/76435416-e2dfd280-63ae-11ea-9e65-6f232ab23da5.png)